.env.base

```diff
@@ -180,3 +180,7 @@ REACT_APP_EXPERIMENTAL_CUSTOM_SEND_NONCE=false
 # Sentry
 
 REACT_APP_SENTRY_DSN_URL=https://c612e7f4ef0637e4add433a2f4683aa8@o4507174990905344.ingest.de.sentry.io/4507174994444368
+
+# Arbitrum Bridge
+
+REACT_APP_FEATURE_ARBITRUM_BRIDGE=false
```

Comment:
nitpick put this feature flag alongside the others

Reply:
9152388d0977ecd1a88ed97f34193fefdbb82f91